sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Watch.java

```diff
@@ -228,6 +253,23 @@ Instant getWatermark() {
         }
         return res;
       }
+
+      @Override
+      public boolean equals(Object o) {
```

Comment:
These base class implementations (toString, equals, hashCode) look similar to the default provided by AutoValue. Is there some important difference? If it's subtle, perhaps mention it in a comment

Reply:
There was no difference, swapped to use autovalue. Its just a remnant of the old way this was written.